src/agents/tracing/processors.py

```diff
@@ -60,7 +60,8 @@ def __init__(
         self.max_delay = max_delay
 
         # Keep a client open for connection pooling across multiple export calls
-        self._client = httpx.Client(timeout=httpx.Timeout(timeout=60, connect=5.0))
+        # use separate connect/read timeouts
+        self._client = httpx.Client(timeout=httpx.Timeout(60.0, connect=5.0))
```

Comment:
This doesn't change anything on the actual behavior, so we don't need this change.